enterprise/views.py

```diff
@@ -656,9 +733,15 @@ def get(self, request, enterprise_uuid, course_id):
 
         enterprise_customer, course, modes = self.get_base_details(enterprise_uuid, course_id)
 
+        # Verify that the request user belongs to the enterprise against the
+        # provided `enterprise_uuid` and if user does not belong to the
+        # enterprise then redirect the user to the ecommerce flow in LMS.
+        enterprise_customer_user = get_enterprise_customer_user(request.user.id, enterprise_uuid)
```

Comment:
I don't think we want this. If I'm reading this correctly, this prevents the landing page flow from being the mechanism for linking the user to the enterprise.